src/ncp/spinel.c

```diff
@@ -1822,6 +1822,18 @@ const char *spinel_prop_key_to_cstr(spinel_prop_key_t prop_key)
         ret = "THREAD_NETWORK_TIME";
         break;
 
+    case SPINEL_PROP_PERFORMANCE_TEST:
```

Comment:
nit: how about adding these in same order as defined in the `spinel.h` file? After `SPINEL_PROP_PARENT_RESPONSE_INFO`...